### PR TITLE
MAB updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Added 'NES' to existing Coastal Pioneer moorings (TOC_L1 column) - this is the only change to these rows.

Added new MAB moorings - confirm refdes per attached doc 1100-00006_Marine_Infrastructure_Vocabulary_and_Reference_Designators_OOI_2024-01-11_ver_2-04.docx and VOCAB_REFDES_2024-01-15.xlsx.  Also used deck drawings & design Ss's on Alfresco (or in Vault: Documentation/Coastal_Pioneer_MAB/3610 Dwgs): 3610-40001-->-40010 and 3610-40101-->40110
[1100-00006_Marine_Infrastructure_Vocabulary_and_Reference_Designators_OOI_2024-01-11_ver_2-04.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/13963208/1100-00006_Marine_Infrastructure_Vocabulary_and_Reference_Designators_OOI_2024-01-11_ver_2-04.xlsx)
[VOCAB_REFDES_2024-01-15.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/13963211/VOCAB_REFDES_2024-01-15.xlsx)

UPDATED reference files (as of 1/17 2:35pm):
[VOCAB_REFDES_2024-01-17.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/13967926/VOCAB_REFDES_2024-01-17.xlsx)
[1100-00006_Marine_Infrastructure_Vocabulary_and_Reference_Designators_OOI_2024-01-17_ver_2-04(1).xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/13967927/1100-00006_Marine_Infrastructure_Vocabulary_and_Reference_Designators_OOI_2024-01-17_ver_2-04.1.xlsx)

### CODE REVIEW

**Review by snwhite — CHANGES_REQUESTED** [review]
The AUV CTD was changed from a Sea-Bird CTD to a Neil Brown CT.  Vendor should be "NBOSI".  Model may just be "NBOSI CT Sensor"
Rest looks good, including correction of tower SPKIR height.
We may want to remove the ADCPU from the NSIF if Al and Andrew are sure we can't get good data from it.

**Review by snwhite — APPROVED** [review]
Change to AUV CTD (NBOSI) and removal of ADCPU from NSIF looks good.

**Comment by rtravis-whoi** [comment]
[Pioneer MAB Array AM [DT-227]](https://trello.com/c/Aqg3aPfT/227-pioneer-mab-array-am-dt-227)

File: vocab/vocab.csv [hunk 1]
```diff
@@ -480,414 +480,717 @@ CE09OSSM-SBD12-00-DCLENG000,Coastal Endurance,Washington Offshore Surface Moorin
...
+CP16MOAS-A6263-06-PARADN000,Coastal Pioneer MAB,Mobile Assets,Coastal AUV 6263,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,600
+CP16MOAS-A6264,Coastal Pioneer MAB,Mobile Assets,Coastal AUV 6264,,,,0,600
+CP16MOAS-A6264-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Coastal AUV 6264,Mobile Asset Controller,Hydroid,AUV,0,600
+CP16MOAS-A6264-01-FLORTN000,Coastal Pioneer MAB,Mobile Assets,Coastal AUV 6264,3-Wavelength Fluorometer,WET Labs,BBFL2HYD,0,600
+CP16MOAS-A6264-02-DOSTAN000,Coastal Pioneer MAB,Mobile Assets,Coastal AUV 6264,Dissolved Oxygen,Aanderaa,Optode 4330,0,600
+CP16MOAS-A6264-03-CTDAVN000,Coastal Pioneer MAB,Mobile Assets,Coastal AUV 6264,CTD,NBOSI,NBOSI CT Sensor,0,600
+CP16MOAS-A6264-04-NUTNRN000,Coastal Pioneer MAB,Mobile Assets,Coastal AUV 6264,Nitrate,Satlantic,SUNA,0,600
+CP16MOAS-A6264-05-ADCPAN000,Coastal Pioneer MAB,Mobile Assets,Coastal AUV 6264,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,600
+CP16MOAS-A6264-06-PARADN000,Coastal Pioneer MAB,Mobile Assets,Coastal AUV 6264,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,600
```
Comment (cooleyky) [inline vocab/vocab.csv:1193]: All updates look good!

**Review by cooleyky — APPROVED** [review]
Everything looks good after updates

**Pull request merged**